pkg/backend/display/ai.go

```diff
@@ -25,18 +26,23 @@ import (
 
 // CopilotErrorSummaryMetadata contains metadata about a Copilot error summary.
 type CopilotErrorSummaryMetadata struct {
-	Summary   string // The summary generated by Copilot.
-	ElapsedMs int64  // The time taken to generate the summary.
+	Summary string // The summary generated by Copilot.
+}
+
+// ExplainFailureLink returns the link that will open Copilot and trigger it to explain the failure on the given
+// permalink.
+func ExplainFailureLink(permalink string) string {
+	return permalink + "?explainFailure"
 }
 
 // copilotEmojiOr returns the emoji for Copilot or an empty string if emojis are disabled.
 // Exposed here for testing.
 func copilotEmojiOr() string {
```

Comment:
```suggestion
func copilotEmoji() string {
```

Sorry for only realizing this now, but we don't really have an option here, so the function name is a little confusing as it is.

Reply:
Good catch w/ the comments, renamed and added a hopefully better explanation.